build.gradle

```diff
@@ -42,7 +45,8 @@ task validateDbConfig {
     description 'Ensure that the target database properties file exists'
     group 'Verification'
 
-    ext.dbConfig = file('src/main/resources/POSTGRES.properties')
+    def fileName = System.getProperty("geexport.db.info", "db-info.properties")
+    ext.dbConfig = file("src/main/resources/${fileName}")
```

Comment:
Ah i see, however it does seem weird since it is coupled with the source code. It would be more reasonable to do the check in source code and throw exception there. Ultimately if a binary is distributed, there would be no `validateDbConfig` task for end user.